lib/rules/no-synchronous-tests.js

```diff
@@ -16,54 +15,105 @@ module.exports = function (context) {
         'afterEach'
     ];
 
-    function getCalleeName(callee) {
-        if (callee.type === 'MemberExpression') {
-             return callee.object.name + '.' + callee.property.name;
-        }
-
-        return callee.name;
+function getCalleeName(callee) {
```

Comment:
I know this function was already there before but it would be nice to re-use the [`getNodeName`](https://github.com/lo1tuma/eslint-plugin-mocha/blob/0e54998108f86bae7ddb029aae06be6d128fc652/lib/util/ast.js#L17) method from our AST utils.

💅